### PR TITLE
Material and semester2 dates

### PR DESCRIPTION
**Pull request by Romain-Thomas-Shef**

### CODE REVIEW

File: pages/training/fair4rs/index.md [hunk 1]
```diff
@@ -86,10 +89,13 @@ improving your understanding of working with branches and how to make your commi
 to understand pull requests and Git history which in turn makes it easier to collaborate and work on code with others
 (including your future self!). **Date to be decided, it will be during second semester**
```
Comment (tdjames1) [inline pages/training/fair4rs/index.md]:
```suggestion
(including your future self!).
```

File: pages/training/fair4rs/index.md [hunk 2]
```diff
@@ -75,9 +78,9 @@ then we introduce some useful tools and get started with some basic workflow usi
 foundations with collaborative exercises that introduce key concepts such as forks, pull requests and branches and give
 you the chance to get some hands-on experience with using version control in a research setting.
 
-**Monday 18th & Tuesday 19th November 2024 (morning only), In person, [registration link](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/d4d09f67-097b-4451-8ddb-86cb90636c06)** (Another session will be given during second
-semester)
+**Monday 18th & Tuesday 19th November 2024 (morning only), In person.**
 
+**Monday 20th & Tuesday 21st January 2025 (Mornings only), In person, [Registration link]().**
```
Comment (tdjames1) [inline pages/training/fair4rs/index.md]:
```suggestion
**Monday 20th & Tuesday 21st January 2025 (Mornings only), In person, [Registration link](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/d4d09f67-097b-4451-8ddb-86cb90636c06).**
```

**Review by tdjames1 — CHANGES_REQUESTED** [review]
Couple of changes, added Git registration link and removed note about scheduling.

**Comment by Romain-Thomas-Shef** [comment]
Thank you @tdjames1 for the review!

**Review by tdjames1 — APPROVED** [review]
Looks good to go!

File: pages/training/fair4rs/index.md [hunk 3]
```diff
@@ -75,21 +78,24 @@ then we introduce some useful tools and get started with some basic workflow usi
 foundations with collaborative exercises that introduce key concepts such as forks, pull requests and branches and give
 you the chance to get some hands-on experience with using version control in a research setting.
 
-**Monday 18th & Tuesday 19th November 2024 (morning only), In person, [registration link](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/d4d09f67-097b-4451-8ddb-86cb90636c06)** (Another session will be given during second
-semester)
+**Monday 18th & Tuesday 19th November 2024 (morning only), In person.**
```
Comment (ns-rse) [inline pages/training/fair4rs/index.md:81]: Are we intentionally leaving a history of past sessions?

If not I'll remove this.

**Pull request merged**